homeassistant/components/nibe_heatpump/sensor.py

```diff
@@ -19,6 +19,7 @@
     ENERGY_KILO_WATT_HOUR,
     ENERGY_MEGA_WATT_HOUR,
     ENERGY_WATT_HOUR,
+    POWER_KILO_WATT,
```

Comment:
Can you use UnitOf enjm?

Reply:
I can do that in a non patch release. I need a lib update anyway.